packages/tailwindcss/src/scss/_container.scss

```diff
@@ -0,0 +1,24 @@
+.hn {
+  &-container {
+    &-header {
+      @apply grid;
+      @apply grid-cols-4 sm:grid-cols-8 md:grid-cols-12 lg:grid-cols-12 xl:grid-cols-12;
+      @apply gap-5 sm:gap-4 lg:gap-6 xl:gap-7;
+      @apply mx-5 sm:mx-4 lg:mx-6 xl:mx-auto;
+      @apply xl:w-[1920px];
```

Comment:
hello bro @hn-backend-juntapao , also same here bro can you also put this into variables. thanks

Reply:
for line 8 only hehe @hn-backend-juntapao